airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/GenerateInputActivityImpl.java

```diff
@@ -80,6 +86,15 @@ public GeneratedJobInput getSyncWorkflowInput(final SyncInput input) {
 
       final JobRunConfig jobRunConfig = TemporalWorkflowUtils.createJobRunConfig(jobId, attempt);
 
+      List<StandardDestinationDefinition> destinationDefinitionList = configRepository.listStandardDestinationDefinitions(true);
```

Comment:
Do we have the destinationDefinitionId at this point? If we have that it may be easier / more reliable to get the definition directly by the ID.

Reply:
I also tried to find the destination definition id and use the `getStandardDestinationDefinition` method. But in this case, destination definition id doesn't transfer and I can't get it here. Maybe I missed something.